Android/app/src/main/res/layout/activity_mmiw_ar.xml

```diff
@@ -26,4 +26,59 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent" />
 
+    <Button
+        android:id="@+id/back_button"
+        android:layout_width="46dp"
+        android:layout_height="46dp"
+        android:layout_marginStart="36dp"
+        android:layout_marginBottom="8dp"
+        android:background="@drawable/ic_back"
+        android:labelFor="@string/back_to_camera"
+        app:layout_constraintBottom_toTopOf="@+id/back_button_text_view"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <TextView
+        android:id="@+id/back_button_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
+        android:text="@string/back"
```

Comment:
I bet you can add the labelFor attribute to this textView on the button and it should stop complaining

Reply:
Nuts, I wish I had seen this. I found labelFor just by typing `label` and seeing the suggestions. Your explanation was much better than the one random page I found on the web...